### PR TITLE
Fleet: Add synthetics integration

### PR DESCRIPTION
**Pull request by EamonnTP**
### Summary

This PR adds content for the Synthetics integration as an extra step to the Quick start topic. 

### Docs preview

https://observability-docs_636.docs-preview.app.elstc.co/guide/en/fleet/master/fleet-quick-start.html#add-synthetics-integration
### Related issue

Closes https://github.com/elastic/observability-docs/issues/519

### CODE REVIEW

**Review by dedemorton — COMMENTED** [review]
Looks great! Just a couple of minor comments. I tested the HTTP monitor, and the steps worked as expected.

File: docs/en/ingest-management/getting-started.asciidoc [hunk 1]
```diff
@@ -119,16 +121,95 @@ Then navigate to the [Metrics System] Host overview dashboard:
...
+
+| *ICMP* | Uses an ICMP `v4` and `v6` Echo Request to ping the configured hosts. By default,
+the host name is required.
+
+For detailed information about ICMP options, see our
+{heartbeat-ref}/monitor-icmp-options.html[Heartbeat] documentation.
+
+|===
+
+. Enter the monitor URL and select a monitor interval in seconds or minutes. By default, a monitoring
```
[the remark this answers is not in the document]
Comment (dedemorton) [reply docs/en/ingest-management/getting-started.asciidoc:183]: Maybe just URL you want to monitor for availability?

File: docs/en/ingest-management/getting-started.asciidoc [hunk 2]
```diff
@@ -119,16 +121,95 @@ Then navigate to the [Metrics System] Host overview dashboard:
...
++
+[role="screenshot"]
+image::images/kibana-fleet-integrations-synthetics.png[{fleet} showing Synthetics integration]
+. Click the Elastic Synthetics integration to see more details about it, then click
+**Add Elastic Synthetics**.
++
+[role="screenshot"]
+image::images/kibana-fleet-integrations-synthetics-overview.png[{fleet} showing Synthetics integration overview]
+
+. On the **Add Elastic Synthetics** page, select the `default` policy.
```
[the remark this answers is not in the document]
Comment (EamonnTP) [reply docs/en/ingest-management/getting-started.asciidoc:149]: Ah! Thanks.

File: docs/en/ingest-management/getting-started.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (EamonnTP) [reply docs/en/ingest-management/getting-started.asciidoc:183]: Great suggestion. Thanks.

**Review by dedemorton — APPROVED** [review]
LGTM!

**Review by paulb-elastic — APPROVED** [review]
LGTM

**Pull request merged**